.github/workflows/build.yaml

```diff
@@ -206,12 +206,32 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    env:
+      file: TEST
+
     steps:
     - uses: actions/checkout@v4
 
     - name: Download all artifacts
       uses: actions/download-artifact@v4
 
+    - name: Set file name
+      if: startsWith(github.ref, 'refs/tags/')
+      run: echo "file=${{ github.ref_name }}" >> "$GITHUB_ENV"
+
+    - name: Rename artifacts
+      run: |
+        mv vcpkg-x64-*-${{ github.sha }}/vcpkg-x64-windows-2019-${{ github.sha }}.7z vcpkg-x64-2019-${{ env.file }}.7z
```

Comment:
We should still have `windows` in the filename. MacOS and Android dependencies have always been a nightmare, so might make sense to move to vcpkg, and can both have what Microsoft call x64 as the architecture, so it'd only be the 2019 and 2022 hinting that it's Windows if we don't say Windows.

Reply:
Done. In a way that still prevents the double "windows" elsid didn't like before.